qemu/tests/cfg/rng_read_longtime.cfg

```diff
@@ -21,7 +21,7 @@
             driver_check_cmd = "WIN_UTILS:\devcon\wnet_amd64\devcon.exe status @DRIVER_ID"
     Linux:
         session_cmd_timeout = 360
-        read_rng_cmd = "dd if=/dev/random bs=4 count=100 2>/dev/null|hexdump"
+        read_rng_cmd = "dd if=/dev/random bs=4 count=10000 2>/dev/null|hexdump"
```

Comment:
What's the reason to change `count` to 10000 please?

Reply:
100 is used before as it's slow in amd host, now /dev/urandom is update to /dev/random, 10000 is  acceptabled in amd host.

rng read is slow in amd host quick in intel host before, which cause the test failed. 10000 is ok for both now